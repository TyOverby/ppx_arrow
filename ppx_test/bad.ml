let k = [%arrow (fun _ -> i)]
let s = [%arrow (fun () -> i)]
