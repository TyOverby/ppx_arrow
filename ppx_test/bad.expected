[1mFile "bad.ml", line 1, characters 8-29[0m:
1 | let k = [%arrow (fun _ -> i)]
            [1;31m^^^^^^^^^^^^^^^^^^^^^[0m
[1;31mError[0m: only plain identifier patterns are allowed for the wrapping function for an arrow-expression
